presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionConnectorConfig.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.property;
+
+import com.facebook.airlift.configuration.Config;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.units.DataSize;
+
+import java.util.Map;
+
+public class NativeExecutionConnectorConfig
+{
+    private static final String CACHE_ENABLED = "cache.enabled";
+    private static final String CACHE_MAX_CACHE_SIZE = "cache.max-cache-size";
+    private static final String CONNECTOR_NAME = "connector.name";
```

Comment:
Could you point me where these configs are defined in C++ code?

Reply:
You can search these names In PrestoServer.cpp. Kind of clunky, needing some refactoring. But currently it works this way :)

Reply:
Please add comment to refer to the path of `PrestoServer.cpp` especially we are in the same repo now